common/lexer/flex_lexer_adapter.h

```diff
@@ -107,6 +113,21 @@ class FlexLexerAdapter : private CodeStreamHolder, protected L, public Lexer {
     }
   }
 
+  // Skip n characters from the begining of the token
+  void YYSkip(size_t skip) {
+    last_token_.text = absl::string_view(
+        last_token_.text.begin() + skip, last_token_.text.length() - skip);
+  }
+
+  // Discard n characters from the end of the token
+  void YYDiscard(size_t discard) {
+    last_token_.text =
+      absl::string_view(last_token_.text.begin(),
+                        last_token_.text.length() - discard);
```

Comment:
Same as:

https://github.com/abseil/abseil-cpp/blob/1a02b7a2054c24f900dab796edb812f9260b51a6/absl/strings/string_view.h#L341

Maybe name this `DiscardSuffix` to make it clearer what is discarded?

Reply:
Right, changed the implementation and renamed, thanks!